dprint.json

```diff
@@ -0,0 +1,18 @@
+{
+	"lineWidth": 120,
+	"useTabs": true,
```

Comment:
Everyone can adjust the view to the needs - 2 space, 4, even 8 😉 
https://www.kindacode.com/article/vs-code-how-to-change-indentation-2-spaces-4-spaces/
https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-personal-account-settings/managing-your-tab-size-rendering-preference

Reply:
I'm always in favour of tabs for that exact reason. I'll ask the team to see if there is strong objection.